### ISSUE TITLE
Security: Upgrade to jQuery UI 1.13.0

### ISSUE THREAD

**Issue by avoinea** [collective/collective.js.jqueryui#46]
![2021-12-18-164238_729x427_scrot](https://user-images.githubusercontent.com/323385/146645003-72af970c-423c-41b9-a247-66853208de8f.png)

**Comment by wesleybl** [comment]
I tried following the update steps described in:

https://github.com/collective/collective.js.jqueryui/blob/master/docs/UPGRADE.txt

However, the `jqueryui` 1.13.1 download no longer makes the code separate from the plugins available. See:

https://jqueryui.com/download

This hampers the choice plugins functionality, described in:

https://github.com/collective/collective.js.jqueryui#include-plugins-and-optimizations

One option would be to clone the jqueryui repository and build it to generate the files. But that makes things more complicated.

So I ask if we can remove this functionality, or if it is really important.

@mauritsvanrees @agitator @avoinea @toutpt @maethu

**Comment by mauritsvanrees** [comment]
Being able to slim the javascript down is a useful option. But upstream has made this impossible/difficult, then we may need to remove this. Would need be a a major breaking release I would say.
But I don't actively use this package.

**Comment by maethu** [comment]
I agree. If removed it simply should be a new major release with a hint in the README.

**Comment by szakitibi** [comment]
> I agree. If removed it simply should be a new major release with a hint in the README.

Something like this?
jQueryUI 1.13.2 - https://github.com/szakitibi/collective.js.jqueryui

**Comment by wesleybl** [comment]
@szakitibi would you like to contribute to the `collective.js.jqueryui`? Please open a issue at:

https://github.com/collective/Collective/issues

and ask to add to the collective organization. So you can create a branch here in this repository.

**Comment by szakitibi** [comment]
> and ask to add to the collective organization. So you can create a branch here in this repository.

https://github.com/collective/Collective/issues/1785#issuecomment-1564638425 :D

Well to avoid complications I simply created the pull request see #56.
